src/components/EnterBarcodePage/CompleteBarcodeButton.tsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import Button from "@material-ui/core/Button";
+import { makeStyles } from "@material-ui/core/styles";
+
+const CompleteBarcodeButton = ({
```

Comment:
same goes for this, right now our approach is to create this same button for every submit flow which is super redundant, but for the sake of getting these changes merged, we'll save that for later